lessons/jest.md

```diff
@@ -0,0 +1,89 @@
+# ModernJS - Jest
+## Welcome to Modern JS
+
+## Jest
+In this lesson we'll be learning about [Jest](https://facebook.github.io/jest) which is an easy and powerful tool for testing your javascript projects.
+
+### What is Jest?
+It is a unit testing framework, creating with the idea of being a "zero configuration" testing experience.
+We'll spend less time setting up test runners, assertion libraries, mocking tools and more time writing tests, leading to more stable and healthy code bases. 
+
+### Features
+- Automatically generating snapshots for snapshot testing
+- Running tests in parallel, meaning it's super performant
+- Sandboxing tests and automatically tearing down global state so you don't have to worry about conflicts
+- Code coverage reports
+- Built-in mocking and assertions
+
```

Comment:
Is it worth talking about watch mode too? Pretty simple but I really like that for TDD stuff where it'll re run tests as you change your code? No worries if not but I like it 😬